src/main/java/io/kaitai/struct/visualizer/VisualizerPanel.java

```diff
@@ -159,10 +152,10 @@ public void valueChanged(TreeSelectionEvent event) {
                 if (node.posStart() == null || node.posEnd() == null)
                     return;
                 HexLibSelectionModel select = hexEditor.getSelectionModel();
-                ArrayList<Point> intervals = new ArrayList<Point>();
+                ArrayList<Point> intervals = new ArrayList<>();
```

Comment:
Using this kind of syntax requires Java 8, I guess. Probably worth update maven file to reflect that dependency?

Reply:
No, diamond operator [introduced in Java 7](https://en.wikipedia.org/wiki/Generics_in_Java#Diamond_operator)